class LearnException
{
public static void main(String[] args)
{

int a=10;
int b=0;
int d=10;

try{
//System.out.println("1");

int c=a/b;
System.out.println(d);

//System.out.println("2");

}
/*catch(ArithmeticException ae)
{
System.out.println("Sorry for the inconvenience");*/

//System.out.println(ae);
//throw ae;

//System.out.println(ae);

}
//System.out.println(c);
}
}
}